Library/Homebrew/cmd/update-report.rb

```diff
@@ -45,7 +47,7 @@ def update_report
     end
 
     if hub.reporters.empty?
-      puts "Already up-to-date."
+      puts "Homebrew is already up-to-date."
```

Comment:
Makes sense, particularly given that this affects a lot more users than the presumably few that default to verbose output. And the previous ambiguity as to what this refers to should be gone, now that there's a title printed above (in verbose mode only). Thanks for staying critical!